### ISSUE TITLE
I got your email,  but it's implications are obscure

### ISSUE THREAD

**Issue by davecb** [keybase/keybase-issues#3574]
I got:
`Someone - probably you - used "Recover account" option on Keybase.

Since you have active devices or paper keys, you can't reset your passphrase. The following devices still have access to your account and have power to set a new passphrase or provision another device:

    quick hurt (a paper key; you can use this to provision a new computer)
    miles

In case you don't remember, here's your Keybase username: davecb. `

It sounds bad, as I didn't reset my passphrase lately. In fact, I forgot I had keybase.
I don't see instructions on resetting my passphrase anywhere (!)

Could you perhaps rewrite the email warning to say
- why It's important?
- what to do next?

**Comment by junderw** [comment]
Anyone can "use Recover account" and it's the equivalent of "Forgot your password" on other sites, all someone needs is your username and they can click "Recover Account" and it will send you that email.

> "Since you have active devices or paper keys, you can't reset your passphrase."

This is telling you (assuming you clicked the button to recover) "hey, you can't recover your account because your account security settings are set to high."

> "The following devices..."

This is just saying "hey in case you forgot how keybase works, you have devices which can recover your account for you, but our keybase servers can not. oh and in case you forgot your username here it is as well."

So...

tl;dr "Someone clicked "recover account" but even if they hacked your email and could read the email keybase sent, basically they are just saying "hey, our recovery system can't do anything for you, so we'll just remind you of how you can get yourself out of this situation (assuming you actually lost your password etc)"

aka nothing to worry.
